evaluation/data/ims/fv3_mapping/README.md

```diff
@@ -0,0 +1,26 @@
+(1) Code to create file with indexes used to map from ASCII IMS file onto FV3 grid. 
+
+Must be run for each FV3 resolution  - need to change resolution (2 places) in create_fv3_mapping.f90. 
+
+Input (hard-coded into create_fv3_mapping.f90): 
```

Comment:
is this still relevant? are these really hard coded?

Reply:
No, I modified it via nml file. I removed this paragraph.